scanner/src/main/kotlin/scanners/scancode/ScanCode.kt

```diff
@@ -162,37 +152,17 @@ class ScanCode(
         val url = "https://github.com/nexB/scancode-toolkit/archive/$archive"
 
         log.info { "Downloading $scannerName from $url... " }
+        val unpackDir = createOrtTempDir(expectedVersion).apply { deleteOnExit() }
```

Comment:
Any reason not to squash this with the previous commit?

Reply:
I just had a mental split between "bootstrapping scanners" and "bootstrapping anything else" in mind. I also would have trouble finding a common "prefix" for the commit message 😉 But I can squash if you prefer.

Reply:
Yes, that's why I asked, because you did the other scanners in the previous commit. But I don't care too much so I'll approve.